src/game/building/farmersInn.js

```diff
@@ -0,0 +1,4 @@
+import { compose } from 'redux'
+import { actionOption } from '../common/player'
+
+export default state => compose(actionOption('farmersInn'))(state)
```

Comment:
a fun trick to convert this to point-free style...

```javascript
export default compose(actionOption('farmersInn'))
```

and then since it's only composing one function

```javascript
export default actionOption('farmersInn')
```

no need to change, as what you have written might be easier to understand... i just thought i would share :)

Reply:
Nope, I've changed it, looks cleaner. Thanks for the pointer.